lib/streamlit/runtime/memory_session_storage.py

```diff
@@ -36,7 +36,7 @@ class MemorySessionStorage(SessionStorage):
     def __init__(
         self,
         maxsize: int = 128,
-        ttl_seconds: int = 5 * 60,  # 5 minutes
+        ttl_seconds: int = 2 * 60,  # 2 minutes
```

Comment:
Dropping this from 5->2 minutes since it makes me a bit nervous that `TTLCache` only completely evicts entries so that the gc can harvest them after a mutating operation (`del`/`set`).

This is probably just paranoia since it seems unlikely that having just 128 sessions worth of extra memory will be enough to cause memory issues in all but the most memory intensive apps, but 2 minutes is also plenty of time to hold sessions when a connection being dropped is just a transient network blip.

Reply:
I could imagine situations where the longer time would be helpful (riding on BART, router connection issues), but without good data on how much memory each session adds, and distribution of connection losses, it's hard to say for sure where the cutoff should be. 2 minutes seems probably fine.

Reply:
Yeah, I think it'd probably be best to see if we can collect some data on the distribution of session memory usage (if this is possible/allowed) before making any decisions on trying to improve reconnect behavior past short network blips that only last a few seconds max. In the BART/router connection scenarios, it's annoying that you lose your state, but at least there's a relatively obvious reason as to why.